src/screens/ConsentScreen.tsx

```diff
@@ -100,7 +105,14 @@ export const ConsentScreen = ({
         </View>
       </ScrollView>
       <View style={styles.buttonsContainer}>
-        {acceptText && <Text style={styles.acceptText}>{acceptText}</Text>}
+        {acceptanceItems?.map(
```

Comment:
Should we do a slice here to just grab the first 2?

Reply:
My original thought is since we limit the consent builder to have a max of 2 acceptance items, we should be safe on the mobile side of things. IF we ever decide to increase that limit, we'll update the consent builder, and we shouldn't have to modify the SDK, and bump the SDK version in the client's mobile app.

Reply:
No, but most of users (if not all) use the consent builder.

Reply:
I made a new commit to limit to 2 acceptance items max.